cypress/e2e/tests/custom-metrics/analysis_metrics.test.ts

```diff
@@ -28,88 +28,90 @@ const metricName = "konveyor_tasks_initiated_total";
-            metrics.getValue(metricName).then((counterValue) => {
-                counter = counterValue;
-            });
-            Application.open(true);
+    beforeEach("Load data and define interceptors", function () {
+        cy.fixture("application").then(function (appData) {
+            this.appData = appData;
         });
-
-        it("Perform analyses - Validate the tasks initiated counter increased", function () {
-            // For source code analysis application must have source code URL git or svn
-            const bookServerApp = new Analysis(
-                getRandomApplicationData("bookserverApp", {
-                    sourceData: this.appData["bookserver-app"],
-                }),
-                getRandomAnalysisData(this.analysisData["source_analysis_on_bookserverapp"])
-            );
-            bookServerApp.create();
-            cy.wait("@getApplication");
-            cy.wait(2000);
-            bookServerApp.analyze();
-            bookServerApp.verifyAnalysisStatus("Completed");
-            counter++;
-            applicationList.push(bookServerApp);
-
-            Application.open(true);
-            const application = new Analysis(
-                getRandomApplicationData("uploadBinary"),
-                getRandomAnalysisData(this.analysisData["uploadbinary_analysis_on_acmeair"])
-            );
-            application.create();
-            cy.wait("@getApplication");
-            cy.wait(2000);
-            application.analyze();
-            application.verifyAnalysisStatus("Completed");
-            counter++;
-            applicationList.push(application);
-
-            // Validate the tasks initiated counter increased
-            metrics.validateMetric(metricName, counter);
+        cy.fixture("analysis").then(function (analysisData) {
+            this.analysisData = analysisData;
         });
 
-        it("Perform analysis on tackle-testapp without credentials - Validate analysis failed but counter increased", function () {
-            // For tackle test app source credentials are required.
-            const tackleTestApp = new Analysis(
-                getRandomApplicationData("tackle-testapp", {
-                    sourceData: this.appData["tackle-testapp-git"],
-                }),
-                getRandomAnalysisData(this.analysisData["source+dep_analysis_on_tackletestapp"])
-            );
-            tackleTestApp.create();
-            cy.wait("@getApplication");
-            cy.wait(2000);
-            tackleTestApp.analyze();
-            tackleTestApp.verifyAnalysisStatus("Failed");
-
-            counter++;
-            applicationList.push(tackleTestApp);
-
-            // Validate the tasks initiated counter increased
-            metrics.validateMetric(metricName, counter);
-        });
+        cy.intercept("GET", "/hub/application*").as("getApplication");
 
-        it("Delete applications - Validate the tasks initiated counter didn't change", function () {
```

Comment:
Does deleting application changes the counter ? If yes then let's cover this TC.

Reply:
Delete application does not change the counter and here we validate this
https://github.com/konveyor/tackle-ui-tests/pull/1184/files/920511fe81899897873e049755343594ffea3f2a#diff-fd4355ecd7f3b89fb885e61883545a32dfa51982fd8cbbad4fe8001e1a3d3be7R113